parsons/newmode/newmode.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, api_user=None, api_password=None, api_version=None):
         Returns:
             Newmode class
         """
+        logger.warning("Newmode V1 API will be sunset in Feburary 2025.")
```

Comment:
definitely need more robust language here, but am hoping to discuss with Shauna along with the Parsons deprecation/versioning process

Reply:
This sounds pretty good, actually, but once we figure out the approach we might want to provide guidance here.